config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
+KEY_PASS=KEY_FILE
+AUTH_TESSEL_OA2="https://client_key@auth.tessel.io"
```

Comment:
Since third parties won't usually have access to the deployed Tessel OAuth server, why not default this to the development seed value: `AUTH_TESSEL_OA2=http://tessel-proxy-dev:the-tessel-devenv@localhost:3002`?